spec/models/ability_spec.rb

```diff
@@ -25,6 +25,107 @@
         it { expect(ability).not_to be_able_to(:show, other_user) }
       end
 
+      describe 'who is allowed to see email address in user profile' do
+
+        # email address is hidden: admin, user's supervisor in current season (confirmed)
+        # email address is not hidden: admin, confirmed user, user him/herself
+
+        let(:other_user) { FactoryGirl.create(:user) }
+
+        context 'when email address is hidden' do
+          context 'when an admin' do
+            before do
+              allow(user).to receive(:admin?).and_return(true)
+              allow(ability).to receive(:supervises?).with(other_user, user).and_return(false)
+            end
+            it 'allows to see hidden email address' do
+              other_user.hide_email = true
+              expect(ability).to be_able_to(:read_email, other_user)
+            end
+          end
+
+          context 'when user\'s supervisor in current season (confirmed)' do
```

Comment:
In all other specs we use single quotes for `describe` and `context`. If I change my single quotes to double quotes, it will look inconsistent. Is it ok? Or should I fix the quotes everywhere? (seems to be a nightmare 😃 ) 
Idk, what do you think?